backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/StubRenderer.kt

```diff
@@ -18,6 +24,8 @@ object StubRenderer {
                 }
                 +"*/"
             }
+            this.descriptor?.extractKDocString()?.let { +it }
```

Comment:
This part should probably go before `this.comment?.contentLines`, because the latter is a `@note` or a `@warning`.

Reply:
Done.

Reply:
(This is again the case)